i18n/en.json

```diff
@@ -600,7 +600,7 @@
   "admin.ldap.emailAttrEx": "E.g.: \"mail\" or \"userPrincipalName\"",
   "admin.ldap.emailAttrTitle": "Email Attribute:",
   "admin.ldap.enableDesc": "When true, Mattermost allows login using AD/LDAP",
-  "admin.ldap.enableSyncDesc": "When true, Mattermost periodically synchronizes users from AD/LDAP. When false, user attributes are updated from SAML during user login.",
+  "admin.ldap.enableSyncDesc": "When true, Mattermost periodically synchronizes users from AD/LDAP. When false, user attributes are updated in Mattermost during user login.",
```

Comment:
While `SAML` is clearly wrong, perhaps this should read, `... are updated from AD/LDAP during user login only.`?

Reply:
@lieut-data Makes sense, updated.